rwth_upper_body_skeleton_random_walk/launch/fixed.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+
+<include file ="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
+<include file ="$(find upper_body_detector)/launch/upper_body_detector.launch"/>
```

Comment:
You can also add it to the `*_robot.launch` file in that packed directly. Maybe with a parameter so one can choose to start the people perception with or without it. In any case, please create the minimal launch file Marc asked for, then I can include it into the overall system launch file afterwards.